src/components/Api.js

```diff
@@ -0,0 +1,130 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._auth = headers.authorization;
+    this._contentType = headers.contentType;
+  }
+
+  getUser() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: { authorization: this._auth },
+    }).then((res) => {
```

Comment:
The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```